src/OpenSage.Game/Game.cs

```diff
@@ -217,6 +217,9 @@ public Scene Scene
 
                     AddComponentsRecursive(_scene.Entities);
                 }
+
+                foreach (var system in GameSystems)
```

Comment:
No problem for this time, but just a style note, please add braces even for single-line control-flow statements.